modules/must-gather-storage-limit.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * support/gathering-cluster-data.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="must-gather-storage-limit_{context}"]
+= Changing the must-gather storage limit
+
+When using the `oc adm must-gather` command to collect data the default maximum storage for the information is 30% of the storage capacity of the container. Once the 30% limit is reached the container is killed and the gathering process stops. Information already gathered will be downloaded to your local storage. You will need either a container with more storage capacity or to adjust the maximum volume percentage before running the must-gather command again.
```

Comment:
* ISG says to use "After" instead of "Once"
* I don't think "killed" is a great term to use, even if it's technically accurate. ISG says to avoid when possible. I'd see if you can replace this with something else if it maintains the same meaning
* For the last sentence, should we add something to be more explicit about why they'd want to fix the storage and re-run it?

```suggestion
When using the `oc adm must-gather` command to collect data, the default maximum storage for the information is 30% of the storage capacity of the container. After the 30% limit is reached, the container is terminated and the gathering process stops. Information already gathered is downloaded to your local storage. To generate a complete must-gather archive, you must either use a container with more storage capacity or to adjust the maximum volume percentage before running the must-gather command again.
```